test/unit/carriers/canada_post_test.rb

```diff
@@ -130,9 +132,4 @@ def test_delivery_range_with_invalid_date
 
     assert_equal [], rate_estimates.rates[0].delivery_range
   end
-
-  def test_line_items_with_nil_values
-    @line_items << Package.new(500, [2, 3, 4], :description => "another box full of stuff", :value => nil)
-    @carrier.find_rates(@origin, @destination, @line_items)
-  end
```

Comment:
This tests looks unfinished to me. The HTTP request is not stubbed out (so it executes a slow HTTP request), and we don't do any assertions, so this test will always pass.

I decided to just remove it.